docs/decisions/0007-adjustments.rst

```diff
@@ -0,0 +1,39 @@
+0007 Ledger Adjustments
+#######################
+
+Status
+******
+**Accepted** (October 2023)
+
+Context
+*******
+We'd like to have the abilitiy to manually adjust the balance of ledgers
+in a somewhat unstructured way - that is, in a way that doesn't reverse
+an existing transaction.
```

Comment:
[suggestion] Would it be helpful to explain in a bit more depth why reversing a transaction isn't always possible, whereby manually adjusting the balance becomes necessary? A bit more business context here may be helpful when we review this ADR in the future.